README.md

```diff
@@ -5,84 +5,120 @@ Sawtooth, allowing users to exchange quantities of customized "Assets" with
+`rest_api/config.py` with sensitive security information and other settings.
+An example file is provided to illustrate what settings are available and how
+they should be used. To get started, you could simply copy this file into the
+appropriate location. Using `bash` from the root project directory, that would
+look like this:
 
 ```bash
-bin/dev-tools -p
+cp rest_api/config.py.example rest_api/config.py
 ```
 
-Next, build client files:
+Note that while this is fine for development purposes, the example secret keys
+provided are _not_ secure. These settings in particular should be changed
+before deployment.
+
+Now with your config file in place, Docker can take care of the rest. From the
+root project directory, simply run:
+
 ```bash
-bin/dev-tools -c
+docker-compose up
 ```
 
-Finally, use `docker-compose` to build and run each component:
+This will take awhile the first time it runs, but when complete will be running
+all required components in separate containers. Many of the components will be
+available through HTTP endpoints, including:
+- The Marketplace REST API will be at **http://localhost:8040**
+- The _SawbuckManager_ web app will be at **http://localhost:8041**.
+- RethinkDB's admin panel will be available at **http://localhost:8042**
+- Sawtooth's blockchain REST API will be available at **http://localhost:8043**
+
+In addition, you will likely want to run the `mktadm` CLI, which provides
+commands for interacting with _SawbuckManager_, like seeding initial data, or
+scheduling daily deals. This is simple using the provided shell container:
 
 ```bash
-docker-compose up
+docker exec -it market-shell bash
 ```
 
-This will create containers for all components with access to the local repo,
-and run them along with the necessary Sawtooth components. Once the process is
-complete a number of HTTP endpoints will be available:
-- The Marketplace REST API will be at **http://localhost:8000**
-- The SawbucksManager web app will be at **http://localhost:3000**.
-- Sawtooth's blockchain REST API will be available at **http://localhost:8008**
-- RethinkDB's admin panel will be available at **http://localhost:9090**
+This will open a bash terminal within the shell container, with the `bin/`
+directory in the PATH, and all dependencies installed. Now to seed
+_SawbuckManager_ with all the needed Offers and Assets run:
+
+```bash
+mktadm seed --url market-rest-api:8000 --data sawbuck_app/app_data.yaml
+```
 
-Note that some dependencies may need to be installed locally, including
-Sawtooth dependencies like the`sawtooth_sdk`, as well as some `pip3` modules.
+And if you would like to schedule daily deals using _cron_, run:
+
+```bash
+mktadm schedule --daily "renew --url market-rest-api:8000 --data sawbuck_app/app_data.yaml"
+```
+
+Note that you _must_ leave this container running for it to submit new deals
+each day. If you want to simply exit the shell (this will not stop the
+container), you can simply run:
+
+```bash
+exit
+```
 
```

Comment:
Some language that in order for changes to the source to take effect you must restart the the docker-compose session(?) and restart it, with a `ctrl-c` and then rerunning the `docker-compose up` command.